components/Pagination/sub-components/ActionButton.jsx

```diff
@@ -15,31 +15,30 @@ const ActionButton = styled.a`
     border 0.2s ease-in-out;
   user-select: none;
   white-space: nowrap;
+  border-radius: 4px;
+  border: 2px solid transparent;
 
   ${({
     theme: {
       colors: {
-        neutral: { 300: neutral300 },
         primary: { 100: primary100, 700: primary700 },
       },
       spacing: { xsmall, xxsmall },
     },
   }) => `
     margin: 0 ${xxsmall}px;
     padding: ${xsmall}px;
+    color: ${primary700};
 
-      &[aria-disabled=true] {
-        color: ${neutral300};
-        cursor: not-allowed;
```

Comment:
Because the first action button, when the first page is checked in pagination, is not displayed instead disabled. The same happen to next button, when the last page is checked.